src/Elasticsearch/Framework/Command/ElasticsearchCleanIndicesCommand.php

```diff
@@ -29,15 +28,6 @@ public function __construct(
         parent::__construct();
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function configure(): void
-    {
-        $this
-            ->addOption('force', 'f', InputOption::VALUE_NONE, 'Do not ask for confirmation');
```

Comment:
we cannot remove that option, we need to deprecate, otherwise the scripts of people will break

Reply:
And if it is released in the next major?

Reply:
Alternatively, I'll change the original check from a layered `if` to an `or` check.

Reply:
i would add an if around if ! Feature::isActive('v6.7.0.0')....

Reply:
@shyim 
I have added a feature check to both the option and its check